### PR TITLE
db changes

### PR DESCRIPTION
**Pull request by salabh-aot**
- added audit fields to all models
 - added `decision_by_id` to `events`

### CODE REVIEW

File: reports-api/migrations/versions/11a3360d6a01_added_audit_fields_to_all_models.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('ea_acts', sa.Column('created_by', sa.String(), nullable=True))
+    op.add_column('ea_acts', sa.Column('created_at', sa.DateTime(),server_default=sa.text('now()'), nullable=True))
+    op.add_column('ea_acts', sa.Column('updated_by', sa.String(), nullable=True))
+    op.add_column('ea_acts', sa.Column('updated_at', sa.DateTime(), nullable=True))
+    op.add_column('ea_acts', sa.Column('is_active', sa.Boolean(), server_default='t', nullable=True))
+    op.add_column('ea_acts', sa.Column('is_deleted', sa.Boolean(), server_default='f', nullable=True))
```
Comment (dinesh-aot) [inline reports-api/migrations/versions/11a3360d6a01_added_audit_fields_to_all_models.py:26]: The default of "is_deleted" should be false.right?

File: reports-api/migrations/versions/11a3360d6a01_added_audit_fields_to_all_models.py [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+    op.add_column('eao_teams', sa.Column('created_by', sa.String(), nullable=True))
+    op.add_column('eao_teams', sa.Column('created_at', sa.DateTime(),server_default=sa.text('now()'), nullable=True))
+    op.add_column('eao_teams', sa.Column('updated_by', sa.String(), nullable=True))
+    op.add_column('eao_teams', sa.Column('updated_at', sa.DateTime(), nullable=True))
+    op.add_column('eao_teams', sa.Column('is_active', sa.Boolean(), server_default='t', nullable=True))
+    op.add_column('eao_teams', sa.Column('is_deleted', sa.Boolean(), server_default='f', nullable=True))
+    op.add_column('events', sa.Column('decision_by_id', sa.Integer(), nullable=True))
+    op.create_foreign_key(None, 'events', 'staffs', ['decision_by_id'], ['id'])
```
Comment (dinesh-aot) [inline reports-api/migrations/versions/11a3360d6a01_added_audit_fields_to_all_models.py:34]: I hope it is okay to have the field to be nullable and assigned as FK at the same time. Some database, this is not allowed.

File: reports-api/migrations/versions/11a3360d6a01_added_audit_fields_to_all_models.py (hunk 1 above)
Comment (salabh-aot) [reply reports-api/migrations/versions/11a3360d6a01_added_audit_fields_to_all_models.py:26]: It is false (`'f'`)

File: reports-api/migrations/versions/11a3360d6a01_added_audit_fields_to_all_models.py (hunk 2 above)
Comment (salabh-aot) [reply reports-api/migrations/versions/11a3360d6a01_added_audit_fields_to_all_models.py:34]: We have nullable foreign keys in other tables also. Have not faced any issues so far.

**Pull request merged**